### ISSUE TITLE
[DOCS] Update example video.

### ISSUE THREAD

**Issue by KendallDoesCoding** [KendallDoesCoding/Choose-Your-Own-Adventure-Game#67]
Add a example video which includes the new GUI, don't remove the existing video as that works as well if GUI is not functional on VM/Computer/Gitpod, etc.

**Comment by immachakata** [comment]
Hi. Is the video recorded and uploaded somewhere? If so, I would love to work on it (share the video url).

**Comment by KendallDoesCoding** [comment]
> Hi. Is the video recorded and uploaded somewhere? If so, I would love to work on it (share the video url).

No, the person working on the issue must make the video as well.

**Comment by Raulj123** [comment]
Can I be assigned this? would the new vid be replaced with the current vid that's in the README.md

**Comment by KendallDoesCoding** [comment]
> Can I be assigned this? would the new vid be replaced with the current vid that's in the README.md

Sure... No, we need both the new video and the old video, because the old video functionality is still there if your computer/VM doesn't accept pygame (Can't load it or smthing).

**Comment by KendallDoesCoding** [comment]
@Raulj123 - any update?

**Comment by Raulj123** [comment]
Just saw I got assigned this I'll work on it later

**Comment by KendallDoesCoding** [comment]
@Raulj123 any update?

**Comment by Raulj123** [comment]
> @Raulj123 any update?

lol it's been 2 years
